lib/suspenders/generators/app_generator.rb

```diff
@@ -29,9 +34,14 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :skip_test, type: :boolean, default: true,
       desc: "Skip Test Unit"
 
+    def initialize(*args)
```

Comment:
I opted for this implementation because this is how rails does it: https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/app_generator.rb#L228